index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portofolio</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="header">
+            <div class="mylogo"><a class="logo">My Logo</a></div>
+            <div class="vector"><img src="assets/vector.svg" alt="vector"></div>
+        </div>
+
+    </header>
+
+    <section class="main">
+        <h1 class="intro">I'm Morteza Glad to See You!</h1>
+        <p class="paragraph"> I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+            to contact me.</p>
+        <div class="connect">LET'S CONNECT</div>
+        <div class="social">
```

Comment:
- [ ] Kindly put The list of icons in a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
the `a` tag will let you connect your social networks with the icons, using href.